src/app/features/high-fee-drawer/high-fee-drawer.tsx

```diff
@@ -17,12 +16,14 @@ export function HighFeeDrawer(props: { learnMoreUrl: string }) {
   }, [isShowingHighFeeConfirmation, setIsShowingHighFeeConfirmation]);
 
   return (
-    <ControlledDrawer
-      icon={<ErrorIcon color="error.label" size="xl" />}
+    <BaseDrawer
```

Comment:
We pass an `ErrorIcon` to this. I removed it as this was the only place using it. 

I'm not sure how to trigger this screen? If I can trigger it I'll re-add it in a cleaner way or check if it's needed

Reply:
Good with me!

Reply:
@markmhendrickson this is the `High Fee` dialog I mentioned to you yesterday. 

@fbwoolf - do you know how to test this properly / how it should work. I tried to refactor it as it was similar to other dialogs, storing open / close in `jotai` state. I added new simpler `useState` handlers but I need to make sure it works. 

You can search for `setIsShowingHighFeeConfirmation` on my branch

Reply:
@kyranjamie  - see this RE HighFee dialog. 

I guess it shows when the fee is considered high but I am lacking knowledge here as to what is considered a high fee. 

e.g. `as a user when I try and transfer X stacks and the fee is Y then show the high fee warning` 

I think it's still working after my changes but I'm not certain so I want to check